all_protos/injective/evm/v1/trace_config.proto

```
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package injective.evm.v1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.54.0)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)